app/DevelopmentConfig.hs

```diff
@@ -109,45 +191,42 @@ createReqBody org = object [ "campuses" .= ([] :: [T.Text]),
                         "dayPreferences" .= ([] :: [T.Text]),
                         "deliveryModes" .= ([] :: [T.Text]),
                         "departmentProps" .= ([] :: [T.Text]),
-                        "direction" .= ("asc" :: T.Text),
-                        "divisions" .= [T.pack "ARTSC"],
+                        "direction" .= directionValue config,
```

Comment:
This shouldn't be a config value, it's really part of the implementation of the FAS parsing. You can leave this hard-coded.